src/App.js

```diff
@@ -147,6 +148,9 @@ class App extends Component {
                 this.props.dispatch(updateVP(newViewport))
               }}
               onHover={this.onHover}
+              // min and max zoom sets the amount map zooms in and out, So that map is not too small or large.
+              maxZoom={11}
+              minZoom={6}
```

Comment:
This is a great add! So easy to zoom too far out then waste time focusing back in. That said, I would drop minZoom a touch (~5.5?) so the full state w/ county lines can be in view at minimum - it was just out of the borders on my MBP. maxZoom will also want to be higher; 14 seems to get the smallest Cook County zipcodes to full screen, so maybe 15 in case more focus is needed?

Reply:
I have changed minMaxZoom levels to (15, 5.5). It can be easily changed if required in the future.

Reply:
Looks good thanks!